lib/library_web/controllers/plugs/require_admin.ex

```diff
@@ -0,0 +1,23 @@
+defmodule LibraryWeb.Plugs.RequireAdmin do
+  import Plug.Conn
+  import Phoenix.Controller
+  alias LibraryWeb.Router.Helpers
+
+  def init(_params) do
+  end
```

Comment:
We can use the `do:` syntax here for one line functions.

Reply:
Since this function doesn't return anything, the one like `do:` syntax breaks the code.